README.md

```diff
@@ -22,8 +22,8 @@ is often one of `scheme`, `chezscheme` or `chezscheme9.5` (depending on the
 version). On a modern desktop machine, this process (including tests)
 should take less than 5 minutes.
 
-Idris 2 is mostly backwards compatible with Idris 1, with some minor
-exceptions. The most notable user visible differences, which might cause Idris
+Idris 2 is mostly backward compatible with Idris 1, with some minor
```

Comment:
Not an error.
According to OED, this is is still a draft addition, and usage varies, both with the suffix 's', and whether or not the adjective should be hyphenated.